crates/re_log_types/src/time_point/time_int.rs

```diff
@@ -68,11 +68,6 @@ impl TimeInt {
     pub fn abs(&self) -> Self {
         Self(self.0.saturating_abs())
     }
-
-    #[inline]
-    pub fn is_timeless(&self) -> bool {
-        self == &Self::BEGINNING
-    }
```

Comment:
Removing this as it sets a very dangerous precedent (doesn't seem to be used though, which is good news).